06-Job-Negotiation-Protocol.md

```diff
@@ -10,215 +10,53 @@ Figure 5.a Job Negotiation Protocol: Flow
-| Field Name            | Data Type        | Description                                                               |
-+-----------------------+------------------+---------------------------------------------------------------------------+
-| request_id            | U32              | Identifier of the original request                                        |
-+-----------------------+------------------+---------------------------------------------------------------------------+
-| new_mining_job_token  | B0_255           | Unique identifier provided by the pool of the job that the Job            |
-|                       |                  | Negotiator has negotiated with the pool. It MAY be the same token as      |
-|                       |                  | CommitMiningJob::mining_job_token if the pool allows to start mining on   |
-|                       |                  | not yet negotiated job. If the token is different from the one in the     |
-|                       |                  | corresponding CommitMiningJob message (irrespective of if the client is   |
-|                       |                  | already mining using the original token), the client MUST send a          |
-|                       |                  | SetCustomMiningJob message on each Mining Protocol client which wishes to |
-|                       |                  | mine using the negotiated job.                                            |
-+-----------------------+------------------+---------------------------------------------------------------------------+
-```
-
-
-### 6.1.6 `CommitMiningJob.Error` (Server->Client)
-```
-+---------------+-----------+------------------------------------------------------------------------------------------+
-| Field Name    | Data Type | Description                                                                              |
-+---------------+-----------+------------------------------------------------------------------------------------------+
-| request_id    | U32       | Identifier of the original request                                                       |
-+---------------+-----------+------------------------------------------------------------------------------------------+
-| error_code    | STR0_255  |                                                                                          |
-+---------------+-----------+------------------------------------------------------------------------------------------+
-| error_details | B0_64K    | Optional data providing further details to given error                                   |
-+---------------+-----------+------------------------------------------------------------------------------------------+
-```
-
-Possible error codes:
-- `invalid-mining-job-token`
-- `invalid-job-param-value-{}` - `{}` is replaced by a particular field name from `CommitMiningJob` message
-
-
-### 6.1.7 `IdentifyTransactions` (Server->Client)
```

Comment:
This should be part of another sub(protocol) about sanity check that should be in another server. So that there are no ways for the server to know which txs are mined by clients inside JNP